charts/s3gw/questions.yaml

```diff
@@ -2,207 +2,214 @@
 questions:
   # General settings
 
-  - variable: useCertManager
-    label: Use cert-manager
-    default: "true"
-    description: "Use cert-manager to provision TLS certificates"
+  - variable: serviceName
+    default: s3gw
+    description: "S3 Service Name"
+    label: "S3 Service Name"
+    required: true
+    type: string
+    group: "General"
+
+  - variable: ingress.enabled
+    default: true
+    description: "Deploy an Ingress (Required for TLS and UI)"
+    label: "Enable Ingress"
+    required: true
     type: boolean
     group: "General"
 
-  - variable: tls.publicDomain.crt
-    show_if: "useCertManager=false"
-    description: "S3 TLS certificate (Public Domain)"
-    label: "S3 TLS certificate (Public Domain)"
+  - variable: publicDomain
+    show_if: ingress.enabled=true
+    default: example.com
```

Comment:
I'd rather prefer the user to be forced to put an explicit value.
I think it is better, for the public domain variables, to require the user to make the choice.